ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -96,12 +103,24 @@ private fun MDDocument(
     document: Document,
     color: Color,
     style: TextStyle,
+    fontSize: TextUnit,
     fontWeight: FontWeight?,
+    fontFamily: FontFamily?,
     textAlign: TextAlign?,
     allowLinks: Boolean,
-    modifier: Modifier = Modifier,
```

Comment:
Note that I've removed the `modifier` all over the place in here... Reason is that, after adding the new properties, I had to add make the composables use multiple lines, and for some unexplained reason, that caused detekt to find a bunch of new issues related to reusing the same `modifier` instance in multiple views, which is disallowed. Since we weren't using them anyway, I thought to just remove them where I could.